about.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>WebDev Summit/about</title>
+    <link rel="stylesheet" href="./assets/css/about.css" />
+  </head>
+  <body>
+    <!-- header section -->
+    <header>
+      <nav>
+        <div class="mobile-menu">
+                    <button class="mobile-menu-button">&#9776;</button>
+                  </div>
+                  <div class="mobile-menu-options">
+                    <button class="mobile-menu-closing-btn">
+                        &times;
+                    </button>
+                    <ul class="mobile-menu-list-ul">
+                        <li class="mobile-menu-list-li"><a href="./index.html">Home</a></li>
+                        <li class="mobile-menu-list-li"><a href="./about.html">About</a></li>
+                    </ul>
+                  </div>
+        <div class="logo">
+          <a href="/Capstone-Project/index.html"
+            ><img src="./assets/imgs/logo.png" alt="Logo"
+          /></a>
+        </div>
+        <div class="menu">
+          <ul class="menu-list-ul">
+            <li class="menu-list-li"><a href="./index.html">Home</a></li>
+            <li class="menu-list-li"><a href="./about.html">About</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <!-- main section -->
+    <main>
+      <section class="hero-section">
+        <h2>
+          Web Dev Creative <br />
+          Summit 2023
+        </h2>
```

Comment:
- [ ] There is a divider below the title that is missing in your design. Please add a divider to your implementation. 👍🏾 

![divider](https://github.com/MustafaSamiShirzad/Capstone-Project/assets/99656687/b02c6588-6b9b-4393-ba27-d0c9219ddaf7)

Reply:
Added A divider

Reply:
- [ ] As suggested by the previous reviewer, kindly add the red line under the event title to follow Figma's design
### Your design :point_down: 
![](https://user-images.githubusercontent.com/99656687/249116830-b02c6588-6b9b-4393-ba27-d0c9219ddaf7.png)
### Figma design :point_down: 
![Screenshot from 2023-06-27 15-01-20](https://github.com/MustafaSamiShirzad/Capstone-Project/assets/91142432/677c590c-eb31-4942-af35-26b548cf7a13)


> * [ ]  There is a divider below the title that is missing in your design. Please add a divider to your implementation. 👍🏾
> 
> ![divider](https://user-images.githubusercontent.com/99656687/249116830-b02c6588-6b9b-4393-ba27-d0c9219ddaf7.png)